index.js

```diff
@@ -10,8 +10,13 @@ const OK = chalk.green('Downloaded:');
 const FROM = chalk.gray('from');
 
 function download(repo, logger) {
-    utils.downloadRepo(repo).then((files) => {
-        files.forEach(file => logger.info(OK, `${file} ${FROM} ${repo}`));
+    utils.downloadRepo(repo).then(({ files, dependencies }) => {
+        if (files && files.length) {
+            files.forEach(file => logger.info(OK, `${file} ${FROM} ${repo}`));
+        }
+        if (dependencies && dependencies.length) {
+            dependencies.forEach(dep => logger.info(OK, `${dep} ${FROM} ${repo}`));
```

Comment:
I was thinking the same thing. I may close this PR for now in effort to decouple the codebase a little bit first. Ideally would like to have a similar log as `npm` where files are displayed nested under the `repos` they're downloaded from.